build/tasks/publish.js

```diff
@@ -106,28 +104,30 @@ gulp.task("npm:ui:readme", function () {
         .pipe(gulp.dest(npmFolder + "/clarity-ui"));
 });
 
-gulp.task("npm:ui", ["npm:ui:deliverables", "npm:ui:sources", "npm:ui:package", "npm:ui:readme"], function () {});
+gulp.task("npm:ui", ["npm:ui:bundles", "npm:ui:sources", "npm:ui:package", "npm:ui:readme"], function () {});
 
 /**
  * Preparing the clarity-angular package
  */
 
 /**
  * The deliverables for clarity-angular are:
- *   - the compiled js files in commonJS format
- *   - the minified JS bundle
+ *   - the compiled js files in es2015 format
```

Comment:
Absolutely. Yes, we are doing ES5 with ES2015 module format only as you guessed. Will update the comments.